client/main.lua

```diff
@@ -1,29 +1,13 @@
-local lockpickCallback = nil
+local function openLockpick(cb)
+    cb(lib.skillCheck(Config.Difficulty))
+end
 
-AddEventHandler('qb-lockpick:client:openLockpick', function(callback)
-    lockpickCallback = callback
-    openLockpick(true)
-end)
+-- Support for basic qb-lockpick
+AddEventHandler('qb-lockpick:client:openLockpick', openLockpick)
 
-RegisterNUICallback('callback', function(data, cb)
-    openLockpick(false)
-    if lockpickCallback then
-        lockpickCallback(data.success)
-    end
-    cb('ok')
-end)
+-- Support for qb-lock forks like NoPixel
+AddEventHandler('kwk-lockpick:client:openLockpick', openLockpick)
 
-RegisterNUICallback('exit', function(_, cb)
-    openLockpick(false)
-    cb('ok')
-end)
-
-openLockpick = function(bool)
-    SetNuiFocus(bool, bool)
-    SendNUIMessage({
-        action = "ui",
-        toggle = bool,
-    })
-    SetCursorLocation(0.5, 0.2)
-    lockpicking = bool
-end
\ No newline at end of file
+exports('StartLockPickCircle', function()
+    return lib.skillCheck(Config.Difficulty)
+end)
```

Comment:
Wouldn't a net event be better here?

Reply:
I just left this like it was originally.